### DISCUSSION TITLE
Matriz de risco PT (Aplicação Android)

### DISCUSSION THREAD

**Discussion by sousajrps** [dssg-pt/covid19pt-data#742] [Montra de Projectos]
Viva,
disponibilizei na play store uma aplicação Android que consome os dados deste repositório, apresentando-os de uma forma sucinta e de fácil leitura. 
A aplicação chama-se `Matriz de Risco PT ` e apresente dados gerais sobre a pandemia e processo de vacinação.
https://play.google.com/store/apps/details?id=com.sousajrps.covid19pt

Obrigado a todos os que contribuem para este repositório.
Um abraço!

**Comment by davipt** [comment]
Olá João. Onde vais buscar os dados para o "quadradinho do costa"?

**Reply by sousajrps** [reply]
Olá Bruno,
o ficheiro `data.csv` tem os valores da incidência e transmissibilidade no país e no continente.

**Reply by davipt** [reply]
Boa.

É que os dados diários são actualizados automaticamente a partir da API da dashboard (por essa razão só estão disponíveis mais tarde que o PDF), e os valores de incidência e Rt são actualizados manualmente apenas quando houver outra actualização, tipicamente só no dia seguinte junto com as vacinas.

Vou adicionar mais um passo ao processo para que esses valores estejam actualizados junto com os outros dados diários.
Até lá tem em atenção que o ultimo dia de dados pode temporariamente ter esses quatro valores em falta.

**Reply by sousajrps** [reply]
Isso é excelente, muito obrigado pela ajuda. 👍
